processor/src/main/java/org/mapstruct/ap/internal/model/source/TargetReference.java

```diff
@@ -142,25 +148,26 @@ public TargetReference build() {
             // 3. @MappingTarget, with
             // 4. or without parameter name.
             String[] targetPropertyNames = segments;
-            List<PropertyEntry> entries = getTargetEntries( resultType, targetPropertyNames, false );
+            List<PropertyEntry> entries = getTargetEntries( resultType, targetPropertyNames );
```

Comment:
You are looking at a diff between my two commits, right now it is the same as it was before there is no change in the invoking of the method and the method itself. If you think that rename is needed, I can rename it to `resolveTargetEntries`.